addons/payment/models/payment_acquirer.py

```diff
@@ -233,7 +233,7 @@ def _compute_journal_id(self):
                     ],
                     limit=1,
                 )
-                if acquirer.id:
+                if acquirer.id and acquirer.provider != 'transfer':
```

Comment:
This is not enough to fix all the custom payment providers.
We should probably make this check hookable and override it from payment_custom.
WDYT?
CC @smetl

Reply:
@simahawk Indeed. It has to stay something generic but not too much restrictive

Reply:
@royle-vietnam gentle ping :)

Reply:
Hi @simahawk ,I'm waiting for your pr to be merged at https://github.com/odoo/odoo/pull/173459
Do you want me to continue editing this pr?

Reply:
@royle-vietnam I don't have time ATM, if you could move on, that would be great :pray:

Reply:
Hi @smetl @simahawk , I fixed it, the solution here is just to search by the values ​​to be created. Please review, thanks